internal/generator/generator.go

```diff
@@ -125,35 +143,106 @@ func Process(options Options) error {
 }
 
 func createBinding(options Options, storedModel *model.ModelInfo) error {
-	currentModel, err := options.CodeGenerator.ParseSource(options.InPath)
-	if err != nil {
-		return err
-	}
+	return pathForEach(options.InPath, func(filePath string) error {
+		if !options.CodeGenerator.IsSourceFile(filePath) {
+			return nil
+		}
 
-	if err = mergeBindingWithModelInfo(currentModel, storedModel); err != nil {
-		return fmt.Errorf("can't merge model information: %s", err)
-	}
+		// clear meta information from the previous creatBinding() call (when processing multiple files at once)
```

Comment:
```suggestion
		// clear meta information from the previous createBinding() call (when processing multiple files at once)
```
If there is time fix me.